Source/Diagnostics/FullDiagnostics.cpp

```diff
@@ -582,6 +583,8 @@ FullDiagnostics::InitializeFieldFunctors (int lev)
 
     // Species index to loop over species that dump rho per species
     int i = 0;
+    // Species index to loop over species that dump number density per species
+    int isp_nd = 0;
```

Comment:
It looks like all of the CI tests are failing because this doesn't get used...must be happening when PULSAR isn't defined? Do we care about that?

Reply:
oooohhh! thank you for catching this!
yes this needs an ifdef PULSAR, so we avoid conflicts with upstream/development